source/index.html.md

```diff
@@ -553,6 +553,67 @@ Status | Response |
 
 
 
+## status
+
+> &gt;&gt;&gt; Script
+
+```python
+[...]
+import requests
+
+driver_id = "12345"
+
+response = requests.get(
+    f'https://api.zego.com/v1/status/?driverId={driver_id}',
+    headers={'Authorization': AUTHORIZATION_CODE},
+)
+
+print(json.loads(response.content))
+
+```
+
+> &lt;&lt;&lt; Output
+
+
+```python
+{
+  "status": "ENABLED",
+  "policyId": "ABC123",
+  "cover_class_of_use": "Carriage of Goods for Hire & Reward",
+  "cover_level": "Third Party (TPO)",
+  "vehicle_registration": "VY7 JC3"
+}
+```
+
+Supply a Policy ID or Driver ID and receive status information about that particular driver. If a user hasn't consented to sharing their information with the requester then the same error message will be received as when the driver is not found.
+
```

Comment:
We should probably enumarate the possible values of `cover_class_of_use` and `cover_level` here, but this looks great

Reply:
I thought about it, but figured if I left it off it would discourage people hardcoding the strings into anything.